qiskit/circuit/instruction.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, name, num_qubits, num_clbits, params):
             num_clbits (int): instruction's clbit width
             params (list[int|float|complex|str|ndarray|list|ParameterExpression]):
                 list of parameters
+            duration (int or float): instruction's duration. Its type indicates its unit:
+                integer means unitless (dt of backend) and float means seconds.
```

Comment:
It might sometimes be preferable to replace `dt` with `sample duration` or `sample time` since the latter requires less contextual knowledge.
```suggestion
                integer means unitless (sample duration of backend) and float means seconds.
```

Reply:
Thank you for your suggestion. I found I need to update these lines more. I've stopped distinguishing units by types in this PR.